docs/modules/ROOT/pages/overview.adoc

```diff
@@ -0,0 +1,66 @@
+= Hazelcast Cloud
+:description: Hazelcast Cloud is a managed cloud service that simplifies Hazelcast cluster deployments, allowing you to focus on building applications for low-latency querying, aggregation, and stateful computations.
+:page-aliases: use-cases.adoc, faq.adoc
+
+{description}
+
+
+image:serverless-app-server.svg[A Hazelcast client application communicating with a Hazelcast Cloud cluster]
+
+Hazelcast Cloud comes in two flavors:
+
+[cols="1a,1a"]
+|===
+|Flavor|Intended applications
+
+|<<serverless, Serverless>>
+|
+- Starter projects
+- Development environments
+- Unpredicable scaling requirements
+- Budget constraints
+
+|<<dedicated, Dedicated>>
+|
+- Explicit resource and configuration requirements
+- Critical production workloads
+|===
```

Comment:
Wouldn't it be good to consult this with Finn - I believe he might come with some benefit-focus terms. Maybe instead of budget constraints it would be better to state one pricing is time bound (per hour) and the other is capacity bound (per GB)? WDYT?